.github/workflows/stale-bot-matrix.yml

```diff
@@ -0,0 +1,84 @@
+# **when?**
+# Triggered by the repos but usually on a schedule to run once a day
+
+name: "Close stale issues and PRs"
+on:
+  workflow_call:
+
+permissions:
+  issues: write
+  pull-requests: write
+
+env:
+  STALE_ISSUE_MSG: "This issue has been marked as Stale because it has been open for 180 days with no activity. If you would like the issue to remain open, please remove the stale label or comment on the issue, or it will be closed in 7 days."
+  STALE_PR_MSG: "This PR has been marked as Stale because it has been open with no activity as of late. If you would like the PR to remain open, please remove the stale label or comment on the PR, or it will be closed in 7 days."
+  CLOSE_ISSUE_MSG: "Although we are closing this issue as stale, it's not gone forever. Issues can be reopened if there is renewed community interest. Just add a comment to notify the maintainers."
+  CLOSE_PR_MSG: "Although we are closing this PR as stale, it can still be reopened to continue development. Just add a comment to notify the maintainers."
+
+jobs:
+  stale-awaiting-response:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v6
+        with:
+          stale-issue-message: ${{ env.STALE_ISSUE_MSG }}
+          stale-pr-message: ${{ env.STALE_PR_MSG }}
+          close-issue-message: ${{ env.CLOSE_ISSUE_MSG }}
+          close-pr-message: ${{ env.CLOSE_PR_MSG }}
+          close-issue-reason: 'not_planned'
+          days-before-stale: 90
+          any-of-labels: 'awaiting_response', 'more_information_needed'
+
+  stale-first-issue:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v6
+        with:
+          stale-issue-message: ${{ env.STALE_ISSUE_MSG }}
+          stale-pr-message: ${{ env.STALE_PR_MSG }}
+          close-issue-message: ${{ env.CLOSE_ISSUE_MSG }}
+          close-pr-message: ${{ env.CLOSE_PR_MSG }}
+          close-issue-reason: 'not_planned'
+          days-before-stale: 360
+          any-of-labels: 'good_first_issue', 'help_wanted'
+
+  stale-tech-debt:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v6
+        with:
+          stale-issue-message: ${{ env.STALE_ISSUE_MSG }}
+          stale-pr-message: ${{ env.STALE_PR_MSG }}
+          close-issue-message: ${{ env.CLOSE_ISSUE_MSG }}
+          close-pr-message: ${{ env.CLOSE_PR_MSG }}
+          close-issue-reason: 'not_planned'
+          days-before-stale: 720
+          any-of-labels: 'tech_debt'
+
+  stale-default:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v6
+        with:
+          stale-issue-message: ${{ env.STALE_ISSUE_MSG }}
+          stale-pr-message: ${{ env.STALE_PR_MSG }}
+          close-issue-message: ${{ env.CLOSE_ISSUE_MSG }}
+          close-pr-message: ${{ env.CLOSE_PR_MSG }}
+          close-issue-reason: 'not_planned'
+          days-before-stale: 180
+          exempt-issue-labels: 'tech_debt', 'good_first_issue', 'help_wanted', 'awaiting_response', 'more_information_needed'
```

Comment:
This should just be 'tech_debt', 'good_first_issue', 'help_wanted' since 'awaiting_response' and 'more_information_needed' have a 90 day limit and should be dealt with before it gets to 180 days.

Reply:
That makes sense. Done!